app/src/main/java/io/github/droidkaigi/confsched/model/Session.java

```diff
@@ -31,9 +34,7 @@
     @SerializedName("description")
     public String description;
 
-    @Column
-    public int speakerId;
-
+    @Column(value = "speakerId", indexed = true)
```

Comment:
This might not what you think :dancer: 

[Column#value](https://github.com/gfx/Android-Orma/blob/master/annotations/src/main/java/com/github/gfx/android/orma/annotation/Column.java#L27-L30) is the column name of the table.

As of konifer/droidkaigi2016 v1.0.0, there is `@Column public int speakerId` in the table; I prefer to use the existing `speakerId` column for the new `@Column Speaker speaker` defined in this PR, instead of defining migration steps. That's it.